tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -721,6 +721,29 @@ module.exports = {
       return indicators
     },
 
+    setSort: async function (column, isDesc = false) {
+      const columnSelector = util.format(this.elements.filesTableHeaderColumn.selector, xpathHelper.buildXpathLiteral(column))
+      await this.useXpath()
+        .waitForElementVisible(columnSelector)
```

Comment:
From nightwatch docs:

> click() will automatically wait for the element to be present (until the specified timeout).

Do we need to wait until the element is visible or is present enough?

Reply:
let's keep "visible" just to be on the safe side...